src/common.spec.ts

```diff
@@ -36,3 +38,66 @@ describe('unpackSDJWT', () => {
     expect(result).toEqual(expectedSDJWT);
   });
 });
+
+describe('packSDJWT', () => {
+  let hasher;
+  let generateSalt;
+
+  beforeAll(() => {
+    hasher = (data) => {
```

Comment:
@ragnika, since these functions are static they can be defined directly inside the `describe` block. No need to use `beforeAll` callback.